src/core/interpreter.cc

```diff
@@ -474,18 +474,26 @@ int RedisLogCommand(lua_State* lua) {
 void* mimalloc_glue(void* ud, void* ptr, size_t osize, size_t nsize) {
   (void)ud;
   if (nsize == 0) {
+    InterpreterManager::tl_stats().used_bytes -= mi_usable_size(ptr);
```

Comment:
Is something off with your merge of main?

Reply:
fixed